src/main/resources/static/css/widget.css

```diff
@@ -162,6 +162,9 @@ h2 {
 .widget {
     max-width: 400px;
 }
+.widget.fallback {
+    padding: 1rem;
```

Comment:
Instead of REM let's use 4VW for the padding as REM is a font size based unit.

Reply:
Done